Picker/Picker.js

```diff
@@ -147,7 +147,6 @@ const PickerBase = kind({
  * @class Picker
  * @memberof agate/Picker
  * @extends agate/Picker.PickerBase
- * @mixes agate/internal/Picker.PickerDecorator
```

Comment:
Why did you remove this part?

Reply:
I think that all components included in internal directory  look private components, so line 150 should be removed but I'll recover the code because it is out of the scope in this ticket.